extension/background.js

```diff
@@ -11,8 +11,12 @@ chrome.commands.onCommand.addListener(function(command) {
 chrome.runtime.onMessage.addListener(
     function(request, sender, sendResponse) {
         console.log("Received tab message: ", request);
-        if(request.command == "activateTab" && sender.tab && activeTabs.indexOf(sender.tab.id) == -1){
-            activeTabs.push(sender.tab.id);
+        if(request.command == "activateTab" && sender.tab){
```

Comment:
Did you mean: `if (request.command == "activateTab" && sender.tab) {`? I used the style that was used in your code before me. If you wish to apply standard code style across all of the files of this extension, it should be done in separeate Pull Request. I'll do if after you merge this PR.